### ISSUE TITLE
Add BLE test

### ISSUE THREAD

**Issue by nikhil-zinjurde-imgtec** [Creatordev/board-test#22]
test_bluetooth.sh only tests standard BT.  This can be extended to include BLE scan and connect or an entirely new test can be added as test_ble.sh

Acceptance Criteria:
* Scan BLE and display BLE devices
* Connect to BLE device
* Disconnect from BLE device
* Basically, do everything what current bluetooth test does

**Comment by Shpinkso** [comment]
Wouldn't we rather have this in boardfarm?

**Comment by nikhil-zinjurde-imgtec** [comment]
Yes absolutely it should be there.  However, the use for board-test is also for Ci40 developers and users out elsewhere to troubleshoot their boards.  We already have a board test for BT, would be good to extend or add a new for BTLE as well
